docs/source/admin/quick_howto/ciab.rst

```diff
@@ -310,3 +310,15 @@ Dante's socks proxy is an optional container that can be used to provide browser
 #. Once the CDN-in-a-box stack has started, use the aforementioned browser to access traffic portal via the socks proxy on the docker host.
 
 .. seealso:: `The official Docker Compose documentation CLI reference <https://docs.docker.com/compose/reference/>`_ for complete instructions on how to pass service definition files to the ``docker-compose`` executable.
+
+Static Subnet
+-------------
+Since docker-compose will randomly create a subnet and it has a chance to conflict with your network environment, using static subnet is a good choice.
+
+.. code-block:: shell
```

Comment:
formatting looks fine, but this needs a caption so it's linkable.

Reply:
No, that's a reference label. If you look in the VNC container section you'll see:

```rst
	.. code-block:: shell
		:caption: CIAB Startup Using Bash Alias

		# From infrastructure/cdn-in-a-box
		alias mydc="docker-compose -f $PWD/docker-compose.yml -f $PWD/optional/docker-compose.vnc.yml"
		docker volume prune -f
		mydc build
		mydc kill
		mydc rm -fv
		mydc up
```
the `:caption:` option specifies the code block caption. There are other code-blocks without captions, but they shouldn't be. (You of course only need to add captions to your contribution; we'll sort the others out in time)

Reply:
Thank you for your explanation. I have fixed it